### PR TITLE
RPRBLND-1833: Collection render no working correct

### PR DESCRIPTION
**Pull request by VascoPi**
### PURPOSE
A stage is created from all the enabled collections, so in case if active camera isn’t there, it doesn’t include into the stage for final render.
Result: null camera is set as active render camera.
Active camera needs to be added to a stage separately.

### EFFECT OF CHANGE
Render collections with expected result and without errors

### TECHNICAL STEPS
Added use_scene_cameras input for depsgraph_objects to be able exclude a camera for final render
Added sync camera object separately for final render

### CODE REVIEW

File: src/hdusd/engine/engine.py [hunk 1]
```diff
@@ -49,8 +49,10 @@ def _export_depsgraph(self, stage, depsgraph, *,
 
         root_prim = stage.GetPseudoRoot()
 
+        use_scene_cameras = self.TYPE!='FINAL'
+
         objects_len = len(depsgraph.objects)
-        for i, obj in enumerate(depsgraph_objects(depsgraph, space_data, use_scene_lights)):
+        for i, obj in enumerate(depsgraph_objects(depsgraph, space_data, use_scene_lights, use_scene_cameras)):
```
Comment (bnagirniak) [inline src/hdusd/engine/engine.py:55]: use_scene_cameras could be always false, so it is not needed

File: src/hdusd/engine/engine.py [hunk 2]
```diff
@@ -62,6 +64,13 @@ def _export_depsgraph(self, stage, depsgraph, *,
             except Exception as e:
                 log.error(e, 'EXCEPTION:', traceback.format_exc())
 
+        # sync a scene camera separately for final render
+        if not use_scene_cameras:
+            try:
```
Comment (bnagirniak) [inline src/hdusd/engine/engine.py:69]: Move this code to overridden FinalEngine._export_depsgraph

**Review by bnagirniak — COMMENTED** [review]
Code improvements

**Review by bnagirniak — APPROVED** [review]
Tested - works good

**Review by DagerD — APPROVED** [review]
Tested. Works ok.

**Pull request merged**